user_portal.php

```diff
@@ -123,6 +123,31 @@
 	</script>';
 }
 
```

Comment:
This js code should be executed only if the option $_configuration['my_courses_view_by_session']
is turn on.